### PR TITLE
M5 Bookstore: styling

### PR DESCRIPTION
**Pull request by ShiroYaksha90**
- Add missing JSX elements to the project (without adding extra functionality)
- the final design should match Zeplin's design (this includes all elements + the same styling)
- Use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/input.js [hunk 1]
```diff
@@ -19,9 +19,7 @@ const Input = () => {
         title,
         author,
         category,
-      })).then((response) => {
-        console.log(response);
-      });
+      }));
     }
     setTitle('');
     setAuthor('');
```
Comment (OLIPLICHE) [inline src/components/input.js:25]: - [x] Hi, In reading the project requirement, I noticed that you forgot to implement the category for each book, also the icon at the top right. Here is what the requirement is saying: 
-  Your final design should match zeplin's design and includes all elements.

- [x] I would advise you to pay attention to the red sign I marked on the original design picture below for you to understand what I mean. Also to implement the categories and the icon.

- **Original design**: 
Capture
- **Your design**
Capture1

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @ShiroYaksha90, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 - Add styling 👍 
 - README file is descriptive 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/input.js (hunk 1 above)
Comment (ShiroYaksha90) [reply src/components/input.js:25]: @OLIPLICHE Thank you for the review I implemented the required changes

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ShiroYaksha90 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] nice design
- [ ] followed zeplin design to detail

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShiroYaksha90** [comment]
@VICTOR-CHUKWU Thank you for the review 😃

**Pull request merged**